apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/CharSequenceContainsContainsNotAssertionsSpec.kt

```diff
@@ -20,9 +24,9 @@ class CharSequenceContainsContainsNotAssertionsSpec : ch.tutteli.atrium.spec.int
 
         private fun contains(plant: Assert<CharSequence>, a: Any, aX: Array<out Any>): Assert<CharSequence> {
             return if (aX.isEmpty()) {
-                plant contains a
+                plant.asExpect().contains(a).asAssert()
             } else {
-                plant contains Values(a, *aX)
+                plant.asExpect().contains(Values(a, *aX)).asAssert()
```

Comment:
this replacement is wrong (uses Values of cc-infix). please undo and check if ReplaceWith needs to be fixed